geomesa-convert/geomesa-convert-text/src/main/scala/org/locationtech/geomesa/convert/text/DelimitedTextConverter.scala

```diff
@@ -80,44 +76,16 @@ class DelimitedTextConverter(format: CSVFormat,
                              val validating: Boolean)
   extends LinesToSimpleFeatureConverter {
 
-  var curString: String = null
-  val q = Queues.newArrayBlockingQueue[String](32)
-  // if the record to write is bigger than the buffer size of the PipedReader
-  // then the writer will block until the reader reads data off of the pipe.
-  // For this reason, we have to separate the reading and writing into two
-  // threads
-  val writer = new PipedWriter()
-  val reader = new PipedReader(writer, options.pipeSize)  // record size
-  val parser = format.parse(reader).iterator()
-  val separator = format.getRecordSeparator
-
-  val es = Executors.newSingleThreadExecutor()
-  es.submit(new Runnable {
-    override def run(): Unit = {
-      while (true) {
-        val s = q.take()
-        // make sure the input is not null and is nonempty...if it is empty the threads will deadlock
-        if (s != null && s.nonEmpty) {
-          writer.write(s)
-          writer.write(separator)
-          writer.flush()
-        }
-      }
-    }
-  })
-
   override def processInput(is: Iterator[String], ec: EvaluationContext): Iterator[SimpleFeature] = {
     ec.counter.incLineCount(options.skipLines)
     super.processInput(is.drop(options.skipLines), ec)
   }
 
   override def fromInputType(string: String): Seq[Array[Any]] = {
-    // empty strings cause deadlock
```

Comment:
i believe i previously fixed empty strings and made a test case...i think maybe this is it: https://github.com/locationtech/geomesa/blob/master/geomesa-convert/geomesa-convert-text/src/test/scala/org/locationtech/geomesa/convert/text/NewLinesTest.scala